packages/react-dom/src/__tests__/ReactDOMSelect-test.js

```diff
@@ -768,14 +952,20 @@ describe('ReactDOMSelect', () => {
       </select>
     );
     const container = document.createElement('div');
-    const node = ReactDOM.render(stub, container);
+    const root = ReactDOMClient.createRoot(container);
+
+    await act(() => {
+      root.render(stub);
+    });
+
+    const node = container.firstChild;
 
     expect(node.options[0].selected).toBe(false); // a
     expect(node.options[1].selected).toBe(true); // b
     expect(node.options[2].selected).toBe(false); // c
   });
 
-  it('should allow controlling `value` in a nested render', () => {
+  it('should allow controlling `value` in a nested legacy render', async () => {
```

Comment:
This test calls `render` using a container that is already owned by the parent root. This pattern is no longer supported with `createRoot` and would error with 
```
Warning: You are calling ReactDOMClient.createRoot() on a container that has already been passed to createRoot() before. Instead, call root.render() on the existing root instead if you want to update it.
```

Seems to me this case is no longer supported with the new root.